bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+
+		<label>MetOffice DataHub Bridge</label>
+		<description>MetOffice DataHub API Bridge</description>
+
+		<properties>
+			<property name="Site Specific API Call Count"/>
+		</properties>
+
+		<config-description>
+			<parameter name="siteSpecificClientId" type="text" required="true">
+				<context>password</context>
+				<label>Site Specific Subscription Client Id</label>
```

Comment:
```suggestion
				<label>Subscription Client Id</label>
```

Your not going to like this i guess, but adjust all labels in this XML-file to:
_<25 chars, max 2-3 words and properly capitalized_

When finished, don't forget to run the i18n plugin.

Reply:
Adjusted while attempting to preserve some context on the labels.